x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/trial_license_complete_tier/import_rules.ts

```diff
@@ -1376,6 +1378,65 @@ export default ({ getService }: FtrProviderContext): void => {
           });
         });
 
+        it('should resolve 100 or more exception references when importing into a clean slate', async () => {
+          // So importing a rule that references an exception list
+          // Keep in mind, no exception lists or rules exist yet
+          const rules = range(150).map((idx) =>
+            getCustomQueryRuleParams({
+              rule_id: `${idx}`,
+              exceptions_list: [
+                {
+                  id: `${idx}`,
+                  list_id: `${idx}`,
+                  type: 'detection',
+                  namespace_type: 'single',
+                },
+              ],
+            })
+          );
+          const exceptionLists = range(150).map((idx) => ({
+            ...getImportExceptionsListSchemaMock(`${idx}`),
+            id: `${idx}`,
+            type: 'detection',
+            namespace_type: 'single',
+          }));
+          const exceptionItems = range(150).map((idx) => ({
+            description: 'some description',
+            entries: [
+              {
+                field: 'some.not.nested.field',
+                operator: 'included',
+                type: 'match',
+                value: 'some value',
+              },
+            ],
+            item_id: `item_id_${idx}`,
+            list_id: `${idx}`,
+            name: 'Query with a rule id',
+            type: 'simple',
+          }));
+          const ndjson = combineArraysToNdJson(rules, exceptionLists, exceptionItems);
+
+          // Importing the "simpleRule", along with the exception list
```

Comment:
This comment isn't correct, and IMO should just be deleted. Again, if this was really tricky I would maybe hide it behind a helper like `importRulePayload()`, but in this case it's pretty obvious.